### ISSUE TITLE
[Package]: duck

### ISSUE THREAD

**Issue by jasttw** [termux/termux-packages#15579]
### Why do you want to add this package to Termux?

duck (cyberduck shell) is currently the most reliable cryptomator vault compatible cli application. It is a universal file transfer tool which runs in your shell. Edit files on remote servers, download, upload and copy between servers with FTP, SFTP or WebDAV plus support for cloud storage Amazon S3 & OpenStack Swift deployments.

### Home page URL

https://duck.sh/

### Source code URL

https://github.com/iterate-ch/cyberduck

### Packaging policy acknowledgement

- [X] The requested package complies with [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

It is available in userland Ubuntu if it helps

**Comment by 2096779623** [comment]
This looks to be the first termux package built with the maven !
We probably need a function like `termux_setup_maven` for the termux build system to work.
https://github.com/2096779623/termux-packages/actions/runs/4455307342/jobs/7825044359#step:6:1270

**Comment by jasttw** [comment]
> Does this need root? If so, perhaps add the root label. Does it need multimedia support?

I don't think it needs root or multimedia support to function. It is working normally in userland without any of these

**Comment by Cletip** [comment]
Hello,

Sorry to reopen the conversation like this, but is there any news?

Thank you in advance for your reply.

**Comment by TomJo2000** [comment]
The main issue with this package is that the Linux support is barely existent.

Upstream provides a DEB and RPM package, the only other Linux package for `duck` is from the AUR and that just repacks the RPM.

Documentation also leaves much to be desired, so we're essentially flying blind if we want to package this.
